Extras/Equilibrium/README.md

```diff
@@ -118,72 +118,41 @@ and slippage will be a consideration otherwise, not to mention security.
+    ./Launcher       ftxus MAIN trx/usd  2   1.1 1    PAPER
 
 Or
 
-    cd /home/Extras/Equilibrium
-    ./Launcher ftxus MAIN trx/usd 2 1 1 PAPER
+    cd /home/Equilibrium
+    ./Launcher       ftxus MAIN trx/usd  2   1   1    PAPER
 
-Or
-
-    cd /home/Extras/Equilibrium
-    ./LauncherFuture ftx MAIN TRX-PERP 0.5 Bal AUTO Short
-
-This launches Equilibrium. The arguments are as follows,
-
-    ftxus       Exchange
-    MAIN        Account (Case sensitive)
-    trx/usd     Asset
-    2           Deviation/Take Profit in percent form
-    1 or Bal    Number of lots to buy
-
-        A lot is the minimum position size. if you wanted a $10
-        position of and asset that has a minimum position size of
-        $2.50, you would want 4 lots. 
-
-        Bal load the balance of the (sub)account and converts it into
-        lots based upon the boundary.
-
-    1.1 or AUTO Number of lots to sell
-
-        This is the number of lots to sell. It works exactly like the
-        buy lots with one very distinct difference, if you put
-        **AUTO**, the selling lot size will be 10% more then the
-        buying lot size.
-
-        If your selling lot size is equal to your buying lot size,
-        reverse trading will **NOT** take place.
-
-        If your selling lots size is less then your buying size, you
-        will ACCUMULATE the diffence in your account. For example, if
-        your buying lot size is 2 and your selling lot size is 1.9,
-        you will ACCUMULATE 0.1 of the asset with each SELL.
-
-    Long/Short  Trade direction for futures ONLY. Not used for SPOT.
-
-        Long or Short direction for trading. You can NOT use both in the
-        same (sub)account, doing so WILL cause losses.
+Or, for futures trading
 
-    PAPER
+    cd /home/Equilibrium
+    ./LauncherFuture ftx   MAIN TRX-PERP 0.5 Bal AUTO Short
 
-        This activates the paper mode. Equilibrium will do everything
-        except actually place orders to the exchange.
+This launches Equilibrium. The arguments are as follows:
 
-        **Even in PAPER mode, you API/Secret and Webhook must be
-        correct as Equilibrium WILL login into your account for youe
-        exchange's private API/rate limit values to get the prices.**
+| Argument | Example   | Argument description |
+| --- | --------- | -------------------- |
+| 1 | `ftxus`   | Exchange             |
+| 2 | `MAIN`    | Account (Case sensitive) |
+| 3 | `trx/usd` | Asset                |
+| 4 | `2`       | Deviation/Take Profit/Boundary in percent form. (`2` maps to 2%) |
+| 5 | `1` or `Bal` | **Number of lots to buy** <br><br> A lot is the minimum position size. If you wanted a $10 position of an asset that has a minimum position size of $2.50, you would want `4` lots. <br><br> `Bal`: This tells Equilibrium to calculate the number of lots based on the balance of the (sub)account and the boundary. **NOTE**: This also tells Equilibrium to replace the value for the 6th argument. |
```

Comment:
This looks harder to read here, but check it out over [here](https://github.com/mrsegen/JackrabbitRelay/tree/main/Extras/Equilibrium#usage) in the current fork

Reply:
I like the way the rendered markdown separates out the fields into slightly different colored rows. It seems to make it easier to read.